app/controllers/admin/events/list.js

```diff
@@ -132,7 +132,15 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
       this.notify.success(this.l10n.t('Event has been deleted successfully.'));
     } catch (e) {
       console.warn(e);
-      this.notify.error(this.l10n.t('An unexpected error has occurred.'));
+      if ('errors' in e) {
```

Comment:
Is this even the right way to check? I think you should check for length here

Reply:
@shreyanshdwivedi It works as expected. Also, checking the length of `e.errors` will again crash the whole thing as pointed out by @CosmicCoder96

Reply:
what if errors is not an array. errors[0] will crash.